### PR TITLE
cloud: update roadmap for 202211

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

as title.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/39149

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-roadmap.md [hunk 1]
```diff
@@ -192,19 +115,14 @@ The TiDB Cloud roadmap brings you what's coming in the near future, so you can s
...
-    <td>TiCDC provides cross-region disaster recovery on the cloud.</td>
-    <td>TiCDC provides disaster recovery that ensures data eventual consistency with lower cost on TiDB Cloud.</td>
-  </tr>
-  <tr>
-    <td>Point-in-time recovery (PITR)</td>
-    <td>Support PITR on the cloud.</td>
-    <td>Support cluster-level PITR on the cloud.</td>
+    <td>Backup and Restore</td>
+    <td>✅ Support EBS snapshot-based backup and restore.</td>
+    <td>BR service on TiDB cloud uses EBS snapshot-based backup and recovery.</td>
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-roadmap.md:125]:
```suggestion
    <td>BR service on TiDB Cloud uses EBS snapshot-based backup and restore.</td>
```

File: tidb-cloud/tidb-cloud-roadmap.md [hunk 2]
```diff
@@ -23,128 +23,51 @@ The TiDB Cloud roadmap brings you what's coming in the near future, so you can s
...
+    <td rowspan="3">Developer experience</td>
+    <td>✅ Load sample datasets manually.</td>
     <td>Support loading sample datasets into a cluster. You can use this data to quickly get started with testing the features of TiDB Cloud.</td>
   </tr>
   <tr>
     <td>Add SQL Editor.</td>
     <td>Write and run SQL queries, and view the results in the TiDB console.</td>
   </tr>
+  <tr>
+    <td>Data API</td>
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-roadmap.md:35]: To make wording and format style consistent with other rows.
```suggestion
    <td>Support Data API.</td>
```

**Review by lilin90 — COMMENTED** [review]
Please also add the following info.
"✅: The feature or improvement is already available in TiDB."

**Review by TomShawn — APPROVED** [review]

**Pull request merged**